internal/versionsapi/version.go

```diff
@@ -146,15 +154,16 @@ func (v Version) ListPath(gran Granularity) string {
 
 // ArtifactsURL returns the URL to the artifacts stored for this version.
 // The URL points to a directory.
-func (v Version) ArtifactsURL() string {
-	return constants.CDNRepositoryURL + "/" + v.ArtifactPath()
+func (v Version) ArtifactsURL(apiVersion string) string {
```

Comment:
Can we defined constants (and a custom type) for `apiVersion`?
This would prevent misuse of inputs to the `ArtifactsURL` and `ArtifactPath` functions

Reply:
That's a reasonable idea. I'll try this.